pymatviz/ptable.py

```diff
@@ -703,293 +747,154 @@ def histogram(
 
 
 def ptable_heatmap(
-    values: ElemValues,
+    data,
     *,
-    log: bool | Normalize = False,
-    ax: plt.Axes | None = None,
-    count_mode: ElemCountMode = ElemCountMode.composition,
+    # TODO: to be removed
+    count_mode: ElemCountMode = ElemCountMode.composition,  # separate data preprocess stage
+    # Heatmap specific
+    heat_mode: Literal["value", "fraction", "percent"] | None = "value",
+    # infty_color: str = "lightskyblue",
+    # na_color: str = "white",
+    # zero_color: str = "#eff",  # light gray  # TODO:covered by colormap?
+    # zero_symbol: str | float = "-",
```

Comment:
Not sure if we need to treat zeros specially here (especially for the color, which should be covered by the colormap?)?

Reply:
I would fold this for now as this would be included in the special value handler plan (i.e. select custom color for a particular value, NaN/inf/zero and more).